<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Introduction to rnaseqDegy • rnaseqDegy</title>
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><link href="../deps/font-awesome-6.5.2/css/all.min.css" rel="stylesheet">
<link href="../deps/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet">
<script src="../deps/headroom-0.11.0/headroom.min.js"></script><script src="../deps/headroom-0.11.0/jQuery.headroom.min.js"></script><script src="../deps/bootstrap-toc-1.0.1/bootstrap-toc.min.js"></script><script src="../deps/clipboard.js-2.0.11/clipboard.min.js"></script><script src="../deps/search-1.0.0/autocomplete.jquery.min.js"></script><script src="../deps/search-1.0.0/fuse.min.js"></script><script src="../deps/search-1.0.0/mark.min.js"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Introduction to rnaseqDegy">
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>


    <nav class="navbar navbar-expand-lg fixed-top bg-primary" data-bs-theme="dark" aria-label="Site navigation"><div class="container">

    <a class="navbar-brand me-2" href="../index.html">rnaseqDegy</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.1.0</small>


    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item"><a class="nav-link" href="../index.html">Getting started</a></li>
<li class="nav-item"><a class="nav-link" href="../reference/index.html">Reference</a></li>
<li class="active nav-item"><a class="nav-link" href="../articles/index.html">Vignettes</a></li>
<li class="nav-item"><a class="nav-link" href="../news/index.html">Changelog</a></li>
      </ul>
<ul class="navbar-nav">
<li class="nav-item"><a class="external-link nav-link" href="https://github.com/ebareke/rnaseqDegy"><span class="fa fab fa-github"></span></a></li>
      </ul>
</div>


  </div>
</nav><div class="container template-article">




<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">

      <h1>Introduction to rnaseqDegy</h1>
                        <h4 data-toc-skip class="author">Eric
Bareke</h4>
            
            <h4 data-toc-skip class="date">2025-02-01</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/ebareke/rnaseqDegy/blob/HEAD/vignettes/rnaseqDegy-intro.Rmd" class="external-link"><code>vignettes/rnaseqDegy-intro.Rmd</code></a></small>
      <div class="d-none name"><code>rnaseqDegy-intro.Rmd</code></div>
    </div>

    
    
<div class="section level2">
<h2 id="overview">Overview<a class="anchor" aria-label="anchor" href="#overview"></a>
</h2>
<p><code>rnaseqDegy</code> provides a <strong>fully automated end-to-end
bulk RNA‑seq pipeline</strong> for human and mouse datasets. Starting
from a raw count matrix and a sample design file, it performs QC,
differential expression (DESeq2/edgeR), enhanced visualization, and
comprehensive functional enrichment (ORA and GSEA using offline
GMTs).</p>
<p>This vignette demonstrates a minimal reproducible workflow.</p>
</div>
<div class="section level2">
<h2 id="installation">Installation<a class="anchor" aria-label="anchor" href="#installation"></a>
</h2>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># Install devtools if not installed</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"devtools"</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Install rnaseqDegy from GitHub</span></span>
<span><span class="fu">devtools</span><span class="fu">::</span><span class="fu"><a href="https://remotes.r-lib.org/reference/install_github.html" class="external-link">install_github</a></span><span class="op">(</span><span class="st">"ebareke/rnaseqDegy"</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="input-files">Input Files<a class="anchor" aria-label="anchor" href="#input-files"></a>
</h2>
<div class="section level3">
<h3 id="count-matrix-tsv">Count matrix (TSV)<a class="anchor" aria-label="anchor" href="#count-matrix-tsv"></a>
</h3>
<p>Must contain: - Column 1: gene IDs (Ensembl or Gene Symbol) -
Remaining columns: integer raw counts</p>
<p>Example:</p>
<pre><code>GeneID   S1   S2   S3   S4
ENSG1    50   23   11   9
ENSG2    140  180  90   76</code></pre>
</div>
<div class="section level3">
<h3 id="sample-design-matrix">Sample design matrix<a class="anchor" aria-label="anchor" href="#sample-design-matrix"></a>
</h3>
<p>Must contain exactly: - <code>Sample</code> - <code>Batch</code> -
<code>Condition</code></p>
<p>Example:</p>
<pre><code>Sample   Batch   Condition
S1       B1      Control
S2       B1      Control
S3       B2      Mutant
S4       B2      Mutant</code></pre>
<p>Example input files are provided in:</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"example_counts.tsv"</span>, package <span class="op">=</span> <span class="st">"rnaseqDegy"</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"example_design.tsv"</span>, package <span class="op">=</span> <span class="st">"rnaseqDegy"</span><span class="op">)</span></span></code></pre></div>
</div>
</div>
<div class="section level2">
<h2 id="quick-start-example">Quick Start Example<a class="anchor" aria-label="anchor" href="#quick-start-example"></a>
</h2>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/ebareke/rnaseqDegy" class="external-link">rnaseqDegy</a></span><span class="op">)</span></span>
<span></span>
<span><span class="va">counts</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"example_counts.tsv"</span>, package <span class="op">=</span> <span class="st">"rnaseqDegy"</span><span class="op">)</span></span>
<span><span class="va">design</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"example_design.tsv"</span>, package <span class="op">=</span> <span class="st">"rnaseqDegy"</span><span class="op">)</span></span>
<span></span>
<span><span class="fu"><a href="../reference/run_rnaseqDegy.html">run_rnaseqDegy</a></span><span class="op">(</span></span>
<span>  counts <span class="op">=</span> <span class="va">counts</span>,</span>
<span>  design <span class="op">=</span> <span class="va">design</span>,</span>
<span>  species <span class="op">=</span> <span class="st">"human"</span>,</span>
<span>  de_method <span class="op">=</span> <span class="st">"DESeq2"</span>,</span>
<span>  condition_reference <span class="op">=</span> <span class="st">"Control"</span>,</span>
<span>  condition_test <span class="op">=</span> <span class="st">"Mutant"</span>,</span>
<span>  batch_correction <span class="op">=</span> <span class="cn">TRUE</span>,</span>
<span>  run_ora <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>  run_gsea <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>  output_dir <span class="op">=</span> <span class="st">"rnaseqDegy_example"</span></span>
<span><span class="op">)</span></span></code></pre></div>
<p>This produces: - PCA plots - Sample correlation heatmap - Top
variable-gene heatmap - Differential expression results - Volcano
plot</p>
<p>Enrichment analyses (ORA/GSEA) can be enabled by setting
<code>run_ora = TRUE</code>, <code>run_gsea = TRUE</code> and providing
GMTs.</p>
</div>
<div class="section level2">
<h2 id="differential-expression-options">Differential Expression Options<a class="anchor" aria-label="anchor" href="#differential-expression-options"></a>
</h2>
<div class="section level3">
<h3 id="choosing-de-method">Choosing DE method<a class="anchor" aria-label="anchor" href="#choosing-de-method"></a>
</h3>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">de_method</span> <span class="op">=</span> <span class="st">"DESeq2"</span>   <span class="co"># default</span></span>
<span><span class="co"># OR</span></span>
<span><span class="va">de_method</span> <span class="op">=</span> <span class="st">"edgeR"</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="adjusting-thresholds">Adjusting thresholds<a class="anchor" aria-label="anchor" href="#adjusting-thresholds"></a>
</h3>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">alpha</span> <span class="op">=</span> <span class="fl">0.05</span>     <span class="co"># FDR threshold</span></span>
<span><span class="va">lfc_cutoff</span> <span class="op">=</span> <span class="fl">1</span>   <span class="co"># abs(log2FC) cutoff</span></span></code></pre></div>
</div>
</div>
<div class="section level2">
<h2 id="enrichment-analysis">Enrichment Analysis<a class="anchor" aria-label="anchor" href="#enrichment-analysis"></a>
</h2>
<div class="section level3">
<h3 id="ora-go-bp-mf-cc">ORA (GO BP / MF / CC)<a class="anchor" aria-label="anchor" href="#ora-go-bp-mf-cc"></a>
</h3>
<p>Enabled by:</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">run_ora</span> <span class="op">=</span> <span class="cn">TRUE</span></span></code></pre></div>
<p>Uses: - <code><a href="https://rdrr.io/pkg/clusterProfiler/man/enrichGO.html" class="external-link">clusterProfiler::enrichGO()</a></code> -
<code>simplify()</code> for redundancy reduction</p>
<p>Outputs: - Dotplot - Barplot - Cnet plot - Emap plot</p>
</div>
<div class="section level3">
<h3 id="gsea-offline-gmt">GSEA (offline GMT)<a class="anchor" aria-label="anchor" href="#gsea-offline-gmt"></a>
</h3>
<p>Requires: - <code>gmt_dir</code>: folder containing GMT files -
<code>gmt_files</code>: character vector of GMT filenames</p>
<p>Example:</p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="../reference/run_rnaseqDegy.html">run_rnaseqDegy</a></span><span class="op">(</span></span>
<span>  <span class="va">...</span>,</span>
<span>  run_gsea <span class="op">=</span> <span class="cn">TRUE</span>,</span>
<span>  gmt_dir <span class="op">=</span> <span class="st">"gmt/"</span>,</span>
<span>  gmt_files <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"HALLMARK.gmt"</span>, <span class="st">"GO_BP.gmt"</span><span class="op">)</span></span>
<span><span class="op">)</span></span></code></pre></div>
<p>Outputs: - Dotplot - Barplot - Cnetplot - Emapplot - Ranked GSEA
results table</p>
</div>
</div>
<div class="section level2">
<h2 id="batch-correction">Batch Correction<a class="anchor" aria-label="anchor" href="#batch-correction"></a>
</h2>
<p>Set:</p>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">batch_correction</span> <span class="op">=</span> <span class="cn">TRUE</span></span></code></pre></div>
<p>Uses: <code><a href="https://rdrr.io/pkg/limma/man/removeBatchEffect.html" class="external-link">limma::removeBatchEffect()</a></code> applied to
VST-transformed data.</p>
<div class="section level3">
<h3 id="pca-before-vs-after-batch-correction">PCA before vs after batch correction<a class="anchor" aria-label="anchor" href="#pca-before-vs-after-batch-correction"></a>
</h3>
<p>Both versions are automatically generated based on the option.</p>
</div>
</div>
<div class="section level2">
<h2 id="correlation-heatmap">Correlation Heatmap<a class="anchor" aria-label="anchor" href="#correlation-heatmap"></a>
</h2>
<p>Correlation method controlled by:</p>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">corr_method</span> <span class="op">=</span> <span class="st">"pearson"</span>  <span class="co"># default</span></span>
<span><span class="co"># or "spearman", "kendall"</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="output-structure">Output Structure<a class="anchor" aria-label="anchor" href="#output-structure"></a>
</h2>
<p>After running, <code>output_dir</code> contains:</p>
<pre><code>QC_PCA_*.pdf/png
QC_sampleCorrelation.pdf/png
QC_topVar500_heatmap.pdf/png
DE_results_all.tsv
normalized_counts.tsv
DE_volcano.pdf/png
ORA_GO_BP_*.tsv + plots
ORA_GO_MF_*.tsv + plots
ORA_GO_CC_*.tsv + plots
GSEA_&lt;gmt&gt;_*.tsv + plots</code></pre>
</div>
<div class="section level2">
<h2 id="session-info">Session Info<a class="anchor" aria-label="anchor" href="#session-info"></a>
</h2>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html" class="external-link">sessionInfo</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="conclusion">Conclusion<a class="anchor" aria-label="anchor" href="#conclusion"></a>
</h2>
<p><code>rnaseqDegy</code> provides a streamlined, powerful, and fully
automated workflow for RNA‑seq differential expression and enrichment
analysis. It is ideal for both HPC batch pipelines and everyday
command‑line usage.</p>
<p>For issues or feature requests, visit: <a href="https://github.com/ebareke/rnaseqDegy/issues" class="external-link uri">https://github.com/ebareke/rnaseqDegy/issues</a></p>
</div>
  </main><aside class="col-md-3"><nav id="toc" aria-label="Table of contents"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p>Developed by Eric Bareke.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.2.0.</p>
</div>

    </footer>
</div>





  </body>
</html>
