<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Advanced rnaseqDegy Workflows • rnaseqDegy</title>
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><link href="../deps/font-awesome-6.5.2/css/all.min.css" rel="stylesheet">
<link href="../deps/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet">
<script src="../deps/headroom-0.11.0/headroom.min.js"></script><script src="../deps/headroom-0.11.0/jQuery.headroom.min.js"></script><script src="../deps/bootstrap-toc-1.0.1/bootstrap-toc.min.js"></script><script src="../deps/clipboard.js-2.0.11/clipboard.min.js"></script><script src="../deps/search-1.0.0/autocomplete.jquery.min.js"></script><script src="../deps/search-1.0.0/fuse.min.js"></script><script src="../deps/search-1.0.0/mark.min.js"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Advanced rnaseqDegy Workflows">
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>


    <nav class="navbar navbar-expand-lg fixed-top bg-primary" data-bs-theme="dark" aria-label="Site navigation"><div class="container">

    <a class="navbar-brand me-2" href="../index.html">rnaseqDegy</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.1.0</small>


    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item"><a class="nav-link" href="../index.html">Getting started</a></li>
<li class="nav-item"><a class="nav-link" href="../reference/index.html">Reference</a></li>
<li class="active nav-item"><a class="nav-link" href="../articles/index.html">Vignettes</a></li>
<li class="nav-item"><a class="nav-link" href="../news/index.html">Changelog</a></li>
      </ul>
<ul class="navbar-nav">
<li class="nav-item"><a class="external-link nav-link" href="https://github.com/ebareke/rnaseqDegy"><span class="fa fab fa-github"></span></a></li>
      </ul>
</div>


  </div>
</nav><div class="container template-article">




<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">

      <h1>Advanced rnaseqDegy Workflows</h1>
                        <h4 data-toc-skip class="author">Eric
Bareke</h4>
            
            <h4 data-toc-skip class="date">2025-02-01</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/ebareke/rnaseqDegy/blob/HEAD/vignettes/rnaseqDegy-advanced-workflows.Rmd" class="external-link"><code>vignettes/rnaseqDegy-advanced-workflows.Rmd</code></a></small>
      <div class="d-none name"><code>rnaseqDegy-advanced-workflows.Rmd</code></div>
    </div>

    
    
<div class="section level2">
<h2 id="overview">Overview<a class="anchor" aria-label="anchor" href="#overview"></a>
</h2>
<p>This vignette shows <strong>advanced usage patterns</strong> of
<code>rnaseqDegy</code>, including:</p>
<ul>
<li>Custom filtering thresholds</li>
<li>Running the pipeline with <strong>edgeR</strong> instead of
DESeq2</li>
<li>Enabling <strong>GO ORA</strong> and <strong>GSEA</strong> with
user-supplied GMTs</li>
<li>Running multiple contrasts</li>
<li>Tips for HPC and scripting</li>
</ul>
<p>We assume you already read the <strong>Introduction</strong>
vignette.</p>
</div>
<div class="section level2">
<h2 id="setup">Setup<a class="anchor" aria-label="anchor" href="#setup"></a>
</h2>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/ebareke/rnaseqDegy" class="external-link">rnaseqDegy</a></span><span class="op">)</span></span>
<span></span>
<span><span class="va">counts</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"example_counts.tsv"</span>, package <span class="op">=</span> <span class="st">"rnaseqDegy"</span><span class="op">)</span></span>
<span><span class="va">design</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"example_design.tsv"</span>, package <span class="op">=</span> <span class="st">"rnaseqDegy"</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="custom-filtering-and-correlation-method">1. Custom Filtering and Correlation Method<a class="anchor" aria-label="anchor" href="#custom-filtering-and-correlation-method"></a>
</h2>
<p>You can relax or tighten the expression filtering criteria and change
the correlation method:</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="../reference/run_rnaseqDegy.html">run_rnaseqDegy</a></span><span class="op">(</span></span>
<span>  counts <span class="op">=</span> <span class="va">counts</span>,</span>
<span>  design <span class="op">=</span> <span class="va">design</span>,</span>
<span>  species <span class="op">=</span> <span class="st">"human"</span>,</span>
<span>  de_method <span class="op">=</span> <span class="st">"DESeq2"</span>,</span>
<span>  condition_reference <span class="op">=</span> <span class="st">"Control"</span>,</span>
<span>  condition_test <span class="op">=</span> <span class="st">"Mutant"</span>,</span>
<span>  filter_min_count <span class="op">=</span> <span class="fl">5</span>,     <span class="co"># instead of default 10</span></span>
<span>  filter_min_prop  <span class="op">=</span> <span class="fl">0.4</span>,   <span class="co"># instead of default 0.5</span></span>
<span>  corr_method      <span class="op">=</span> <span class="st">"spearman"</span>,</span>
<span>  batch_correction <span class="op">=</span> <span class="cn">TRUE</span>,</span>
<span>  run_ora <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>  run_gsea <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>  output_dir <span class="op">=</span> <span class="st">"rnaseqDegy_advanced_filter"</span></span>
<span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="using-edger-as-the-de-engine">2. Using edgeR as the DE Engine<a class="anchor" aria-label="anchor" href="#using-edger-as-the-de-engine"></a>
</h2>
<p>To use <strong>edgeR</strong> for differential expression:</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="../reference/run_rnaseqDegy.html">run_rnaseqDegy</a></span><span class="op">(</span></span>
<span>  counts <span class="op">=</span> <span class="va">counts</span>,</span>
<span>  design <span class="op">=</span> <span class="va">design</span>,</span>
<span>  species <span class="op">=</span> <span class="st">"human"</span>,</span>
<span>  de_method <span class="op">=</span> <span class="st">"edgeR"</span>,</span>
<span>  condition_reference <span class="op">=</span> <span class="st">"Control"</span>,</span>
<span>  condition_test <span class="op">=</span> <span class="st">"Mutant"</span>,</span>
<span>  batch_correction <span class="op">=</span> <span class="cn">TRUE</span>,</span>
<span>  alpha <span class="op">=</span> <span class="fl">0.01</span>,      <span class="co"># stricter FDR cutoff</span></span>
<span>  lfc_cutoff <span class="op">=</span> <span class="fl">1.5</span>,  <span class="co"># higher fold-change threshold</span></span>
<span>  run_ora <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>  run_gsea <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>  output_dir <span class="op">=</span> <span class="st">"rnaseqDegy_edgeR"</span></span>
<span><span class="op">)</span></span></code></pre></div>
<p><code>rnaseqDegy</code> will automatically:</p>
<ul>
<li>Fit a GLM with <code>~ batch + group</code>
</li>
<li>Compute QL F-tests</li>
<li>Harmonize output columns to <code>log2FoldChange</code> and
<code>padj</code>
</li>
</ul>
</div>
<div class="section level2">
<h2 id="running-go-ora-bpmfcc">3. Running GO ORA (BP/MF/CC)<a class="anchor" aria-label="anchor" href="#running-go-ora-bpmfcc"></a>
</h2>
<p>To enable GO ORA, simply set <code>run_ora = TRUE</code>:</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="../reference/run_rnaseqDegy.html">run_rnaseqDegy</a></span><span class="op">(</span></span>
<span>  counts <span class="op">=</span> <span class="va">counts</span>,</span>
<span>  design <span class="op">=</span> <span class="va">design</span>,</span>
<span>  species <span class="op">=</span> <span class="st">"human"</span>,</span>
<span>  de_method <span class="op">=</span> <span class="st">"DESeq2"</span>,</span>
<span>  condition_reference <span class="op">=</span> <span class="st">"Control"</span>,</span>
<span>  condition_test <span class="op">=</span> <span class="st">"Mutant"</span>,</span>
<span>  run_ora <span class="op">=</span> <span class="cn">TRUE</span>,</span>
<span>  run_gsea <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>  output_dir <span class="op">=</span> <span class="st">"rnaseqDegy_ORA"</span></span>
<span><span class="op">)</span></span></code></pre></div>
<p>Internally, the pipeline:</p>
<ul>
<li>Uses all expressed genes as <strong>background</strong>
(universe)</li>
<li>Selects significant DEGs using <code>alpha</code> and
<code>lfc_cutoff</code>
</li>
<li>Maps symbols to <strong>ENTREZ IDs</strong>
</li>
<li>Runs <code><a href="https://rdrr.io/pkg/clusterProfiler/man/enrichGO.html" class="external-link">clusterProfiler::enrichGO()</a></code> for
<strong>BP</strong>, <strong>MF</strong>, and <strong>CC</strong>
</li>
<li>Simplifies terms by semantic similarity
(<code>simplify()</code>)</li>
</ul>
<p>The following files are created:</p>
<ul>
<li>
<code>ORA_GO_BP.tsv</code>, <code>ORA_GO_MF.tsv</code>,
<code>ORA_GO_CC.tsv</code>
</li>
<li><code>ORA_GO_&lt;ONT&gt;_dotplot.pdf/png</code></li>
<li><code>ORA_GO_&lt;ONT&gt;_barplot.pdf/png</code></li>
<li><code>ORA_GO_&lt;ONT&gt;_cnetplot.pdf/png</code></li>
<li><code>ORA_GO_&lt;ONT&gt;_emapplot.pdf/png</code></li>
</ul>
</div>
<div class="section level2">
<h2 id="running-gsea-with-offline-gmt-files">4. Running GSEA with Offline GMT Files<a class="anchor" aria-label="anchor" href="#running-gsea-with-offline-gmt-files"></a>
</h2>
<p>To use GSEA you must provide:</p>
<ul>
<li>
<code>gmt_dir</code>: directory containing GMT files</li>
<li>
<code>gmt_files</code>: character vector of GMT filenames</li>
</ul>
<p>Example GMT layout:</p>
<pre class="text"><code>gmt/
 ├── HALLMARK.v2023.1.Hs.symbols.gmt
 └── C5.GO.BP.v2023.1.Hs.symbols.gmt</code></pre>
<p>Example call:</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="../reference/run_rnaseqDegy.html">run_rnaseqDegy</a></span><span class="op">(</span></span>
<span>  counts <span class="op">=</span> <span class="va">counts</span>,</span>
<span>  design <span class="op">=</span> <span class="va">design</span>,</span>
<span>  species <span class="op">=</span> <span class="st">"human"</span>,</span>
<span>  de_method <span class="op">=</span> <span class="st">"DESeq2"</span>,</span>
<span>  condition_reference <span class="op">=</span> <span class="st">"Control"</span>,</span>
<span>  condition_test <span class="op">=</span> <span class="st">"Mutant"</span>,</span>
<span>  run_ora <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>  run_gsea <span class="op">=</span> <span class="cn">TRUE</span>,</span>
<span>  gmt_dir <span class="op">=</span> <span class="st">"gmt"</span>,</span>
<span>  gmt_files <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>    <span class="st">"HALLMARK.v2023.1.Hs.symbols.gmt"</span>,</span>
<span>    <span class="st">"C5.GO.BP.v2023.1.Hs.symbols.gmt"</span></span>
<span>  <span class="op">)</span>,</span>
<span>  output_dir <span class="op">=</span> <span class="st">"rnaseqDegy_GSEA"</span></span>
<span><span class="op">)</span></span></code></pre></div>
<p>For each GMT file, the pipeline:</p>
<ul>
<li>Ranks all genes by log2FC</li>
<li>Runs <code><a href="https://rdrr.io/pkg/clusterProfiler/man/GSEA.html" class="external-link">clusterProfiler::GSEA()</a></code>
</li>
<li>Writes <code>GSEA_&lt;gmt_name&gt;.tsv</code>
</li>
<li>Creates dotplot, barplot, cnetplot, emapplot</li>
</ul>
<div class="section level3">
<h3 id="focusing-on-specific-terms">Focusing on specific terms<a class="anchor" aria-label="anchor" href="#focusing-on-specific-terms"></a>
</h3>
<p>Optionally, you can restrict plots to a subset of terms using
<code>gmt_terms</code>:</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="../reference/run_rnaseqDegy.html">run_rnaseqDegy</a></span><span class="op">(</span></span>
<span>  counts <span class="op">=</span> <span class="va">counts</span>,</span>
<span>  design <span class="op">=</span> <span class="va">design</span>,</span>
<span>  species <span class="op">=</span> <span class="st">"human"</span>,</span>
<span>  de_method <span class="op">=</span> <span class="st">"DESeq2"</span>,</span>
<span>  condition_reference <span class="op">=</span> <span class="st">"Control"</span>,</span>
<span>  condition_test <span class="op">=</span> <span class="st">"Mutant"</span>,</span>
<span>  run_ora <span class="op">=</span> <span class="cn">TRUE</span>,</span>
<span>  run_gsea <span class="op">=</span> <span class="cn">TRUE</span>,</span>
<span>  gmt_dir <span class="op">=</span> <span class="st">"gmt"</span>,</span>
<span>  gmt_files <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"HALLMARK.v2023.1.Hs.symbols.gmt"</span><span class="op">)</span>,</span>
<span>  gmt_terms <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"HALLMARK_APOPTOSIS"</span>, <span class="st">"HALLMARK_INFLAMMATORY_RESPONSE"</span><span class="op">)</span>,</span>
<span>  output_dir <span class="op">=</span> <span class="st">"rnaseqDegy_focus_terms"</span></span>
<span><span class="op">)</span></span></code></pre></div>
</div>
</div>
<div class="section level2">
<h2 id="multiple-contrasts-in-a-script">5. Multiple Contrasts in a Script<a class="anchor" aria-label="anchor" href="#multiple-contrasts-in-a-script"></a>
</h2>
<p><code>rnaseqDegy</code> focuses on <strong>one contrast per
call</strong>, but you can script multiple contrasts:</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">contrasts</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span></span>
<span>  <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span>ref <span class="op">=</span> <span class="st">"Control"</span>, test <span class="op">=</span> <span class="st">"Mutant"</span><span class="op">)</span>,</span>
<span>  <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span>ref <span class="op">=</span> <span class="st">"Control"</span>, test <span class="op">=</span> <span class="st">"Treatment2"</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="kw">for</span> <span class="op">(</span><span class="va">c</span> <span class="kw">in</span> <span class="va">contrasts</span><span class="op">)</span> <span class="op">{</span></span>
<span>  <span class="va">outdir</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/sprintf.html" class="external-link">sprintf</a></span><span class="op">(</span><span class="st">"rnaseqDegy_%s_vs_%s"</span>, <span class="va">c</span><span class="op">$</span><span class="va">ref</span>, <span class="va">c</span><span class="op">$</span><span class="va">test</span><span class="op">)</span></span>
<span></span>
<span>  <span class="fu"><a href="../reference/run_rnaseqDegy.html">run_rnaseqDegy</a></span><span class="op">(</span></span>
<span>    counts <span class="op">=</span> <span class="va">counts</span>,</span>
<span>    design <span class="op">=</span> <span class="va">design</span>,</span>
<span>    species <span class="op">=</span> <span class="st">"human"</span>,</span>
<span>    de_method <span class="op">=</span> <span class="st">"DESeq2"</span>,</span>
<span>    condition_reference <span class="op">=</span> <span class="va">c</span><span class="op">$</span><span class="va">ref</span>,</span>
<span>    condition_test <span class="op">=</span> <span class="va">c</span><span class="op">$</span><span class="va">test</span>,</span>
<span>    batch_correction <span class="op">=</span> <span class="cn">TRUE</span>,</span>
<span>    run_ora <span class="op">=</span> <span class="cn">TRUE</span>,</span>
<span>    run_gsea <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>    output_dir <span class="op">=</span> <span class="va">outdir</span>,</span>
<span>    prefix <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/sprintf.html" class="external-link">sprintf</a></span><span class="op">(</span><span class="st">"%s_vs_%s"</span>, <span class="va">c</span><span class="op">$</span><span class="va">ref</span>, <span class="va">c</span><span class="op">$</span><span class="va">test</span><span class="op">)</span></span>
<span>  <span class="op">)</span></span>
<span><span class="op">}</span></span></code></pre></div>
<p>This pattern is particularly useful on <strong>HPC clusters</strong>,
where each contrast can be launched as a separate job.</p>
</div>
<div class="section level2">
<h2 id="tips-for-hpc-and-automation">6. Tips for HPC and Automation<a class="anchor" aria-label="anchor" href="#tips-for-hpc-and-automation"></a>
</h2>
<ul>
<li>Use absolute paths for <code>counts</code>, <code>design</code>,
<code>gmt_dir</code>, and <code>output_dir</code>.</li>
<li>Keep <code>output_dir</code> unique for each job/contrast.</li>
<li>You can wrap <code><a href="../reference/run_rnaseqDegy.html">run_rnaseqDegy()</a></code> inside a small R script
and call it with <code>Rscript</code> from a scheduler (SLURM, PBS,
etc.).</li>
</ul>
<p>Example SLURM command (shell-level):</p>
<div class="sourceCode" id="cb9"><pre class="sourceCode bash"><code class="sourceCode bash"><span id="cb9-1"><a href="#cb9-1" tabindex="-1"></a><span class="ex">Rscript</span> <span class="at">-e</span> <span class="st">"rnaseqDegy::run_rnaseqDegy(</span><span class="dt">\</span></span>
<span id="cb9-2"><a href="#cb9-2" tabindex="-1"></a><span class="st">  counts = 'counts.tsv', </span><span class="dt">\</span></span>
<span id="cb9-3"><a href="#cb9-3" tabindex="-1"></a><span class="st">  design = 'design.tsv', </span><span class="dt">\</span></span>
<span id="cb9-4"><a href="#cb9-4" tabindex="-1"></a><span class="st">  species = 'human', </span><span class="dt">\</span></span>
<span id="cb9-5"><a href="#cb9-5" tabindex="-1"></a><span class="st">  de_method = 'DESeq2', </span><span class="dt">\</span></span>
<span id="cb9-6"><a href="#cb9-6" tabindex="-1"></a><span class="st">  condition_reference = 'Control', </span><span class="dt">\</span></span>
<span id="cb9-7"><a href="#cb9-7" tabindex="-1"></a><span class="st">  condition_test = 'Mutant', </span><span class="dt">\</span></span>
<span id="cb9-8"><a href="#cb9-8" tabindex="-1"></a><span class="st">  batch_correction = TRUE, </span><span class="dt">\</span></span>
<span id="cb9-9"><a href="#cb9-9" tabindex="-1"></a><span class="st">  run_ora = TRUE, </span><span class="dt">\</span></span>
<span id="cb9-10"><a href="#cb9-10" tabindex="-1"></a><span class="st">  run_gsea = FALSE, </span><span class="dt">\</span></span>
<span id="cb9-11"><a href="#cb9-11" tabindex="-1"></a><span class="st">  output_dir = 'rnaseqDegy_job1')"</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="troubleshooting">7. Troubleshooting<a class="anchor" aria-label="anchor" href="#troubleshooting"></a>
</h2>
<ul>
<li>
<strong>Installation issues</strong>: Ensure that all Bioconductor
dependencies (<code>DESeq2</code>, <code>edgeR</code>,
<code>clusterProfiler</code>, etc.) are installed with
<code><a href="https://bioconductor.github.io/BiocManager/reference/install.html" class="external-link">BiocManager::install()</a></code>.</li>
<li>
<strong>No significant DEGs</strong>: Try relaxing
<code>alpha</code> or <code>lfc_cutoff</code>, or verify your design and
sample size.</li>
<li>
<strong>ORA/GSEA empty</strong>: Ensure enough DEGs for ORA (e.g., ≥
10) and that GMT identifiers match your gene identifiers (symbols).</li>
</ul>
</div>
<div class="section level2">
<h2 id="conclusion">Conclusion<a class="anchor" aria-label="anchor" href="#conclusion"></a>
</h2>
<p>This vignette illustrated advanced control over filtering, DE
engines, enrichment options, and multi-contrast workflows. Combined with
the introductory vignette, you should now be able to integrate
<code>rnaseqDegy</code> into both interactive analyses and production
pipelines.</p>
<p>For questions and feature requests, please open an issue at: <a href="https://github.com/ebareke/rnaseqDegy/issues" class="external-link uri">https://github.com/ebareke/rnaseqDegy/issues</a></p>
</div>
  </main><aside class="col-md-3"><nav id="toc" aria-label="Table of contents"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p>Developed by Eric Bareke.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.2.0.</p>
</div>

    </footer>
</div>





  </body>
</html>
